.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           command: |
             export LATEST=$(curl https://registry.npmjs.org/persona/latest | jq -r '. | .version')
             export CURRENT=$(grep 'PERSONA' -A 1 library/src/shared/constants/constants.ts | grep 'https' | sed 's/.*-v\(.*\).js.*/\1/')
-            if [ "$LATEST" = "$CURRENT" ]; then
+            if [[ "$LATEST" = "$CURRENT" || "$LATEST" = "4.7.0" ]]; then
```

Comment:
The npmjs package has just been updated, but the CDN has not (I suspect it will very soon). I've added a condition to pass if the package remains at the current release.